nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -693,6 +693,9 @@ def exec(self):
         self.output_config.fullExplanation = True  # assign true for this query - it is always ok to compare its results
         self.output_config.configName = os.path.basename(self.config.name) if self.config.name.startswith('./') else \
             self.config.name
+        # if dns entry peers exist but no istio policies are configured,
+        # then actually istio layer exists implicitly, connections to these peers will be considered with the
+        # default Istio outbound traffic mode - allow any
```

Comment:
add an issue about this - to consider later if this should be changed or not, maybe consider a warning about using default outbound traffic mode in the absence of sidecar resource with explicit value for this.

Reply:
done #500